modules/ossm-validating-operators.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-validating-operators_{context}"]
+= Validating Operator installation
+
+//The Operator installation steps include verifying the Operator status in the OpenShift console.
+
+When you install the {ProductName} Operators, OpenShift automatically creates the following objects as part of a successful Operator installation:
+
+* ConfigMaps
+* CustomResourceDefinitions
+* Deployments
+* Pods
+* ReplicaSets
+* Roles
+* RoleBindings
+* Secrets
+* Service Accounts
+* Services
+
+.From the Openshift console
+
+You can verify the Operator pods are available and running in the OpenShift Console.
+
+. Navigate to *Workloads* -> *Pods*.
+. Select the `openshift-operators` namespace.
+. Verify that the following Pods exist and have a status of `running`.
```

Comment:
I think it's actually better to call out that `running` is a literal status due to the OCP guidelines for formatting, otherwise it should be ".. exist and are running." without the backticks.
Pods should be "pods" here though, as per the API formatting guidelines.